### DISCUSSION TITLE
Expo

### DISCUSSION THREAD

**Discussion by KBarreto23** [mockingbot/react-native-zip-archive#269] [Ideas]
Is possible add support to expo?

**Comment by plrthink** [comment]
Yes, it can be used with an expo app, but you should change the project settings. There's some guide shows how to integrate react-native native modules with expo like this https://blog.chiragsrvstv.dev/install-native-modules-with-expo.

**Reply by KBarreto23** [reply]
Thank you, I will try

**Reply by plrthink** [reply]
Hey, do you integrate the package with the expo successfully? Many folks have the same issue as you. So it would be great if you could share the solution.

**Reply by KBarreto23** [reply]
I didn't make it
